e2e/nomostest/testing/wrapper.go

```diff
@@ -45,15 +46,12 @@ func NewShared(t NTB) *Wrapper {
 		}
 	}
 
-	tw := &Wrapper{
-		t: t,
-	}
-	return tw
+	return t
```

Comment:
No need to return the original input argument given that we're not wrapping it anymore.